docs/manual/common/guide/production/ReactivePlatform.md

```diff
@@ -1,39 +1,14 @@
 # Lightbend Reactive Platform
 
-We recommend that you use [Lightbend Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform) to take advantage of the [Split Brain Resolver](http://doc.akka.io/docs/akka/akka-commercial-addons-1.0/java/split-brain-resolver.html) and the [Diagnostics Recorder](http://doc.akka.io/docs/akka/akka-commercial-addons-1.0/common/diagnostics-recorder.html).
+ [Lightbend Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform) is an application development solution powered by an open source core and commercial Production Suite for building scalable Reactive systems on the JVM. The open source components can be used via public repositories as normal, while **the commercial ones are in a secure repository requiring Lightbend-provided credentials**.
```

Comment:
Minor editing suggestions:

[Lightbend Reactive Platform](https://www.lightbend.com/products/lightbend-reactive-platform) is an application development solution powered by an open source core and commercial [Enterprise Suite](https://www.lightbend.com/platform/production) for building Reactive systems on the JVM. The open source components can be used via public repositories as normal, while **Enterprise Suite components are in a secure repository requiring Lightbend-provided credentials**.